From cf33efc0d41a81c67511954fe75b7f63048d86a9 Mon Sep 17 00:00:00 2001
From: Shahab Vahedi <shahab@synopsys.com>
Date: Tue, 10 Dec 2019 16:25:08 +0100
Subject: [PATCH 14/20] gdb: Use correct feature in tdesc-regs for ARC

tdesc-regs.exp test fails for ARC because the test is not
using the correct XML files as target description.  With
this change, the correct directory and files are used.

gdb/testsuite/ChangeLog:
2020-04-01  Shahab Vahedi  <shahab@synopsys.com>

	* gdb.xml/tdesc-regs.exp: Use correct core-regs for ARC.

Will be a part of GDB 11:
https://sourceware.org/git?p=binutils-gdb.git;a=commit;h=3eccb1c8bfd1f119bbc55bf2821d0e4d76116b67
---
 gdb/testsuite/gdb.xml/tdesc-regs.exp |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/gdb/testsuite/gdb.xml/tdesc-regs.exp
+++ b/gdb/testsuite/gdb.xml/tdesc-regs.exp
@@ -32,7 +32,8 @@
     }
     "arc*-*-*" {
 	set architecture "arc:ARCv2"
-	set core-regs {arc-v2.xml}
+	set regdir "arc/"
+	set core-regs {core-v2.xml aux-v2.xml}
     }
     "arm*-*-*" {
 	set regdir "arm/"
